tests/test_morph.py

```diff
@@ -53,24 +155,26 @@ def test_get_2(self):
 def test_MorphHelper_cache_1(nm_load):
     nodes = Mock()
     nodes.get.side_effect = ['morph-A', 'morph-A', 'morph-B', 'morph-A']
-    test_obj = test_module.MorphHelper('morph-dir', nodes)
-    nm_load.side_effect = Mock
-    morph1 = test_obj.get(1)
-    # should get cached object for 'morph-A'
-    assert test_obj.get(2) is morph1
-    # should get new object ('morph-B')
-    assert test_obj.get(3) is not morph1
-    # 'morph-A' was evicted from cache
-    assert test_obj.get(4) is not morph1
+    with patch.object(test_module.MorphHelper, '_is_biophysical', return_value=True):
+        test_obj = test_module.MorphHelper('morph-dir', nodes)
+        nm_load.side_effect = Mock
+        morph1 = test_obj.get(1)
+        # should get cached object for 'morph-A'
+        assert test_obj.get(2) is morph1
+        # should get new object ('morph-B')
+        assert test_obj.get(3) is not morph1
+        # 'morph-A' was evicted from cache
+        assert test_obj.get(4) is not morph1
 
 
 @patch(test_module.__name__ + '.MORPH_CACHE_SIZE', None)
 @patch('neurom.load_neuron')
 def test_MorphHelper_cache_2(nm_load):
     nodes = Mock()
     nodes.get.side_effect = ['morph-A', 'morph-A']
-    test_obj = test_module.MorphHelper('morph-dir', nodes)
-    nm_load.side_effect = Mock
-    morph1 = test_obj.get(1)
-    # same morphology, but no no caching
-    assert test_obj.get(2) is not morph1
\ No newline at end of file
+    with patch.object(test_module.MorphHelper, '_is_biophysical', return_value=True):
+        test_obj = test_module.MorphHelper('morph-dir', nodes)
+        nm_load.side_effect = Mock
+        morph1 = test_obj.get(1)
```

Comment:
indexing from 0 here also please.

Reply:
good catch !

Reply:
with the mocking this is not a problem but yeah that was not consistent